API/DTOs/PlayerRatingChartDataPointDTO.cs

```diff
@@ -6,62 +8,67 @@ namespace API.DTOs;
 public class PlayerRatingChartDataPointDTO
 {
     /// <summary>
-    /// Match name
+    /// Match name, if applicable
```

Comment:
Probably out of scope for this PR but this application right here is exactly where I would love to use a `MatchCompactDTO` instead of spreading a ton of nullable related properties.

Reply:
Can make an issue and implement after merging maybe?